activity.js

```diff
@@ -9,32 +9,37 @@ class Activity {
     this.id = Date.now();
   };
 
-  countdown(minutes, seconds) {
-    setInterval(function() {
-    if (seconds > 0) {
-      seconds = seconds - 1;
-    } else if (minutes > 0 && seconds == 0) {
-      minutes = minutes - 1;
-      seconds = 59;
-    } else if (minutes == 0 && seconds == 0) {
-      minutes = minutes.toString().padStart(2, '0');
-      seconds = seconds.toString().padStart(2, '0');
-      startTimerBtn.innerText = "COMPLETE!";
-      show(logActivityBtn);
-      return timerDisplay.innerText = "YOU ROCK!";
-    };
-     minutes = minutes.toString().padStart(2, '0');
-     seconds = seconds.toString().padStart(2, '0');
-     timerDisplay.innerText = `${minutes}:${seconds}`}, 1000)
-  };
-
   markComplete() {
     this.completed = true;
   };
 
+    countdown(minutes, seconds) {
+      if (!this.completed){
+        this.markComplete();
+        var interval = setInterval( function() {
+          if (seconds > 0) {
+            seconds = seconds - 1;
+          } else if (minutes > 0 && seconds == 0) {
+            minutes = minutes - 1;
+            seconds = 59;
+          } else if (minutes == 0 && seconds == 0) {
+            minutes = minutes.toString().padStart(2, '0');
+            seconds = seconds.toString().padStart(2, '0');
+            startTimerBtn.innerText = "COMPLETE!";
+            show(logActivityBtn);
+            clearInterval(interval);
```

Comment:
clearInterval to save the day! Thank you Google. While making gifs for our README we ran across a bug that we HAD to stop everything to fix. Much trial and error later, this fixed it. Hallelujah.